numpythia/src/_libnumpythia.pyx

```diff
@@ -62,6 +63,16 @@ PARENTS = HepMC.PARENTS
 CHILDREN = HepMC.CHILDREN
 SIBLINGS = HepMC.PRODUCTION_SIBLINGS
 
+cdef class sphericity:
+    cdef Sphericity sph
+
+    def __cinit__(self):
+        sph = Sphericity(2.0, 2)
```

Comment:
I'm no expert but is there a reason to initialise with these specific values? In any case it would be good to add a docstring to the class definition below, where the 2 parameters will be described briefly.

Reply:
I would pass them in. I also don't know what this is doing, though, I just recommend exactly mimicking the C++ interface.

Reply:
Absolutely. I just wanted to bring the *potential* issue up. If the C++ does this then we should follow, of course.

Reply:
The sphericity class initialized with these values is defined as "linear sphericity", the goal is to eventually have a "Linear Sphericity" class, like in Pythia, I am just trying to get this case to work :)

Reply:
OK. Then better add a comment to the close and resolve this conversation.